clients/cobol-lsp-vscode-extension/snippets.json

```diff
@@ -1982,5 +1982,108 @@
 			"$0"
 		],
 		"scope": "cobol"
-	}
+	},
+    "IDMS-CONTROL SECTION ": {
```

Comment:
I see that some identifiers finish with whitespace before the closing quote. Is it intentional?